composer.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "keboola/processor-vacuum",
+    "description": "Deletes all empty files and folders.",
+    "require": {
+        "PHP": "^7.1",
+        "keboola/php-component": "^4.1"
+    },
+    "require-dev": {
+        "jakub-onderka/php-parallel-lint": "^1.0",
+        "keboola/coding-standard": "^4.0",
+        "keboola/php-temp": "^1.0",
+        "phpstan/phpstan-shim": "^0.9.2",
+        "phpunit/phpunit": "^7.0",
+        "symfony/process": "^4.0",
+        "keboola/datadir-tests": "^1.0"
```

Comment:
Vypadá to že to není releasnutá poslední verze generátoru která používá nové datadir testy https://github.com/keboola/component-generator/commits/master

Reply:
jj, vypadá to tak. asi to zkusím updatnout ručně.